README.md

```diff
@@ -78,3 +78,45 @@ cargo build
 This manual twiddling of environment vars is not ideal but given that the alternative is prefixing `cargo build` with these envs on every `cargo build/run`, for now we have chosen to hardcode these in `config.toml` - that may be revisited in the future depending on local pain and/or evolving `boring` upstream build flows.
 
 Note that the Dockerfiles used to build these vendored `boringssl` builds may be found in the respective vendor directories, and can serve as a reference for the build environment needed to generate FIPS-compliant ztunnel builds.
+
+## Metrics
+
+Ztunnel exposes a variety of metrics, at varying levels of stability.
+
+**Core** metrics are considered stable APIs
+**Unstable** metrics may be changed. This includes removal, semantic changes, and label changes.
+
+### Core metrics
+
+#### Traffic metrics
+
+- Tcp Bytes Sent (`istio_tcp_sent_bytes_total`): This is a `COUNTER` which measures the size of total bytes sent during response in case of a TCP connection.
```

Comment:
Is this standard verbiage? It reads somewhat weird to me `size of total bytes`. Could it be something like `total count of bites sent...` instead. I don't feel super strongly about it. If we want to change it would make sense to change Tcp Bytes Received as well but I won't double comment.

Reply:
copied from https://istio.io/latest/docs/reference/config/metrics/